qa/L0_trace/test.sh

```diff
@@ -897,4 +897,132 @@ kill $SERVER_PID
 wait $SERVER_PID
 set +e
 
+################################################################################
```

Comment:
I think you need to update copyrights to 2024 throughout some of these files as well